README.md

```diff
@@ -37,6 +37,7 @@ custom:
         swaggerFiles?: ['./doc/endpointFromPlugin.json', './doc/iCannotPutThisInHttpEvent.json', './doc/aDefinitionWithoutTypescript.json']
         swaggerPath?: 'string'
         apiKeyName?: 'string'
+        customApiKeysHeader?: 'string[]'
```

Comment:
Also, I apologize that I didn't notice this before, but this should probably be `customApiKeyHeaders` instead, since they are multiple headers each related to an API Key.

Reply:
Or even `apiKeyHeaders`, as `custom` is a bit redundant

Reply:
No problem, whatever I need improvement, let's improve!

Thanks